### DISCUSSION TITLE
[Issues Forms] Link a template to a specific project board in the repository

### DISCUSSION THREAD

**Discussion by jeherve** [community/community#5210] [Projects and Issues]
When creating an issue via an issue form template, it would be nice if the yml template offered an option to automatically add that issue to a specific project board. 

It would help with triaging incoming issues in a repository.

--------

Somewhat related: #4266

**Comment by cryptic-wizard** [comment]
Added pic for clarity.  This would allow new issues to show directly on Kanban boards.

![image](https://user-images.githubusercontent.com/87053379/131225307-3fb01a34-ab22-42a4-afe1-8936b6bb8734.png)

**Reply by RolfBippus** [reply]
Thanks, actually exactly(!) what I expected after finding out about projects.
With multiple repos you cannot leave it to the users to "think about adding a project" each time they add an issue.

The alternative to do it automatically via github-actions (https://github.com/marketplace/actions/create-or-update-project-card) in my eyes is too complicated regarding the necessity for access tokens etc. And doing it here seems rather straight forward.

P.S. The same holds for pull request templates!
